### PR TITLE
[StyleCop]  Add warning fixes generating Breaking Changes 

### PR DESCRIPTION
**Pull request by Aliandi**
*Disclaimer*: To fix some SryleCop warnings we might incur in breaking changes. In this case related to protected properties that might be used by end users.
- Rename protected field to match recommended casing

### CODE REVIEW

File: .NET/Microsoft.Recognizers.Text/Matcher/AcAutomaton.cs [hunk 1]
```diff
@@ -1,98 +1,98 @@
...
-
-namespace Microsoft.Recognizers.Text.Matcher
-{
-    public class AcAutomaton<T> : AbstractMatcher<T>
-    {
-        protected readonly AaNode<T> root = new AaNode<T>();
-
-        public AcAutomaton()
-        {
+﻿using System.Collections.Generic;
```
Comment (ceciliaavila) [inline .NET/Microsoft.Recognizers.Text/Matcher/AcAutomaton.cs]: Please, check **AcAutomaton** and **TrieTree**. This appears to be a line ending change.

**Review by ceciliaavila — CHANGES_REQUESTED** [review]
Please check for line ending problems.

**Review by VictorGrycuk — APPROVED** [review]
LGTM

File: .NET/Microsoft.Recognizers.Text/Matcher/AcAutomaton.cs (hunk 1 above)
Comment (ceciliaavila) [reply .NET/Microsoft.Recognizers.Text/Matcher/AcAutomaton.cs]: Done!

**Review by ceciliaavila — APPROVED** [review]
LGTM

File: .NET/Microsoft.Recognizers.Text.NumberWithUnit/Parsers/BaseCurrencyParser.cs [hunk 2]
```diff
@@ -65,9 +65,9 @@ private static void ResolveText(List<ParseResult> prs, string source, int bias)
 
         private string GetResolutionStr(object value)
         {
-            return Config.CultureInfo != null ?
```
Comment (JuanAr) [inline .NET/Microsoft.Recognizers.Text.NumberWithUnit/Parsers/BaseCurrencyParser.cs]: This is not a breaking change. not that it is incorrect, but it should not be part of this PR.

**Review by JuanAr — CHANGES_REQUESTED** [review]
Extract the non-breaking change.

File: .NET/Microsoft.Recognizers.Text.NumberWithUnit/Parsers/BaseCurrencyParser.cs (hunk 2 above)
Comment (ceciliaavila) [reply .NET/Microsoft.Recognizers.Text.NumberWithUnit/Parsers/BaseCurrencyParser.cs]: Done. I'll fix this file in another PR.
Thanks!

**Review by JuanAr — APPROVED** [review]

**Comment by gasper-az** [comment]
Closed due to MS#1420
